jar-service/src/test/java/com/sixsq/slipstream/user/UserResourceTest.java

```diff
@@ -472,23 +468,23 @@ public void putWithUserState() throws ValidationException,
 		withState.remove();
 	}
 
-	@Test
-	public void systemParameterMerge() throws SlipStreamClientException {
-		Request request = createGetRequest(superUser, otherUser.getName());
-
-		// Pick a category that we know always exists
-		String category = "SlipStream_Support";
-		CookieUtils.addAuthnCookie(request, otherUser.getName(), category);
-
-		Response response = executeRequest(request);
-
-		String denormalized = XmlUtil.denormalize(response.getEntityAsText());
-		User user = (User) SerializationUtil.fromXml(denormalized, User.class);
-
-		UserParameter systemParameter = user
-				.getParameter("slipstream.support.email");
-		assertNotNull(systemParameter);
-	}
+//	@Test
```

Comment:
If this test isn't useful, it should just be deleted, or if it may become useful in the future, then use the `@Ignore` attribute.